src/mame/heathkit/h89.cpp

```diff
@@ -54,6 +54,14 @@
 
 namespace {
 
+// Single Step
+#define LOG_SS    (1U << 1)
+
+#define VERBOSE ( LOG_SS )
+#include "logmacro.h"
+
+#define LOGSS(...)    LOGMASKED(LOG_SS,    __VA_ARGS__)
+
```

Comment:
This is all macro glue – it goes outside the anonymous namespace.  Also, probably a good idea not to leave logging enabled here since it’s in a read handler and hence very hot.